packages/preview/exmllent/0.1.0/parser.typ

```diff
@@ -0,0 +1,198 @@
+#let xml-to-worksheets(xml-path) = {
+  let workbook = xml(xml-path).filter(e => if e.tag == "" { false } else { true }).first()
```

Comment:
This won't work as expected, because `xml` can only read files from the current package. The user of your package should call it and pass the resulting string as an argument to your function.